posthog/api/test/test_signup.py

```diff
@@ -0,0 +1,772 @@
+import datetime
+import uuid
+from typing import cast
+from unittest.mock import patch
+
+import pytest
+import pytz
+from django.core import mail
+from django.utils import timezone
+from rest_framework import status
+
+from posthog.models import Dashboard, Organization, Team, User
+from posthog.models.organization import OrganizationInvite
+from posthog.test.base import APIBaseTest
+from posthog.utils import get_instance_realm
+
+
+class TestSignupAPI(APIBaseTest):
+    @classmethod
+    def setUpTestData(cls):
```

Comment:
Change: We now no longer create any test data (as opposed to just not creating a user account) to test with fresh instances only.